lib/components/Input/input-phone-number/input-phone-number-docs.mdx

```diff
@@ -32,8 +32,6 @@ import { Markdown } from "@storybook/blocks";
 | inputSize            | "sm", "md" ,"lg"                               | "lg"                       | Sets the size of the input field.                                                         |
 | variant              | "outline" ,"fill"                              | "outline"                  | Sets the variant style of the input.                                                      |
 | status               | "neutral" ,"error" ,"success"                  | "neutral"                  | Sets the status style of the input.                                                       |
-| placeholder          | string                                         | "00 0000 0000"             | Placeholder text for the input field.                                                     |
-| formatProps          | object                                         | { format: "## #### ####" } | Props for formatting the input value.                                                     |
```

Comment:
| placeholder          | string                                         |       | Placeholder text for the input field.                                                     |	
| formatProps          | object                                         |  | Props for formatting the input value.                                                     |